contracts/cw3-fixed-multisig/Cargo.toml

```diff
@@ -0,0 +1,31 @@
+[package]
+name = "cw3-fixed-multisig"
+version = "0.2.1"
+authors = ["Ethan Frey <ethanfrey@users.noreply.github.com>"]
+edition = "2018"
+description = "Implementing cw3 with an fixed group multisig"
+license = "Apache-2.0"
+repository = "https://github.com/CosmWasm/cosmwasm-plus"
+homepage = "https://cosmwasm.com"
+documentation = "https://docs.cosmwasm.com"
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[features]
+backtraces = ["cosmwasm-std/backtraces"]
+# use library feature to disable all init/handle/query exports
+library = []
+
+[dependencies]
+cosmwasm-std = { version = "0.10.1", features = ["iterator"] }
```

Comment:
Added task migration to issue https://github.com/CosmWasm/cosmwasm-plus/issues/96

Reply:
Hi, better to do that in another MR in my opinion.

Reply:
Yes, let us merge this first (and the nft contract) before migrating to 0.11.

For contract obligations, I need https://github.com/CosmWasm/cosmwasm-plus/pull/97 and would like this PR to be merged (0.10.x compatible) before end of month, and published as 0.2.2. Once those are merged, let's get the 0.11 branch going and prepare a 0.3.0 release.